### PR TITLE
SetGuardian tx generator + tx saver + tx sender

### PR DESCRIPTION
**Pull request by sstanculeanu**

### CODE REVIEW

File: pendingGuardianRemover/txsSender/txSender/txSender.go [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+		return
+	}
+
+	log.Debug("transaction sent", "hash", hash)
+}
+
+func (sender *txSender) hasPendingGuardian(ctx context.Context) bool {
+	guardianData, err := sender.httpClient.GetGuardianData(ctx, sender.senderAddress)
+	if err != nil {
+		return true // force tx in case of error
```
Comment (AdoAdoAdo) [inline pendingGuardianRemover/txsSender/txSender/txSender.go:126]: :+1:

File: pendingGuardianRemover/txsCreator/config.toml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+[API]
+    NetworkAddress = "https://testnet-api.multiversx.com"
+
+[TxGen]
+    NumOfTransactions = 1000
+    ServiceUID        = "ServiceID"
+    ChainID           = "T"
+    GasPrice          = 1000000000
+    GasLimit          = 300000
```
[the remark this answers is not in the document]
Comment (sstanculeanu) [reply pendingGuardianRemover/txsCreator/config.toml]: right, updated to 500k

**Review by AdoAdoAdo — APPROVED** [review]

File: pendingGuardianRemover/go.mod [hunk 3]
```diff
@@ -0,0 +1,34 @@
+module github.com/multiversx/mx-chain-tools-go/pendingGuardianRemover
+
+go 1.18
+
+require (
+	github.com/multiversx/mx-chain-core-go v1.1.34-0.20230308105332-12c26500c7f7
```
Comment (iulianpascalau) [inline pendingGuardianRemover/go.mod:6]: maybe proper release? Or later?

Comment (sstanculeanu) [reply pendingGuardianRemover/go.mod:6]: proper release will come once we have one for feat/freeze-account

**Pull request merged**